pkg/repro/repro.go

```diff
@@ -357,8 +357,18 @@ func (ctx *context) extractProgBisect(entries []*prog.LogEntry, baseDuration tim
 		return baseDuration + time.Duration(entries/4)*time.Second
 	}
 
+	// First check if replaying the log may crash the kernel at all.
+	ret, err := ctx.testProgs(entries, duration(len(entries)), opts)
```

Comment:
Isn't this equivalent to the check we do in the beginning of bisection?
https://github.com/a-nogikh/syzkaller/blob/acecfa4b2c6dc18af186ecbacfcb46049aaa524d/pkg/repro/repro.go#L678C18-L685

Reply:
We used to (the code you referenced is quite old already), now I return the check back.

Reply:
Right (not sure why github pointed me to that branch).